debian/enterprise-kafka/Dockerfile

```diff
@@ -46,3 +45,5 @@ VOLUME ["/var/lib/${COMPONENT}/data", "/etc/${COMPONENT}/secrets"]
 COPY include/etc/confluent/docker /etc/confluent/docker
 
 CMD ["/etc/confluent/docker/run"]
+
+ONBUILD RUN echo 'WARNING: enterprise-kafka image is now deprecated!. Please use cp-server image.'
```

Comment:
Not sure we want these warnings, let me check with @addisonhuddy.

Reply:
@omkreddy Let's remove the warnings.

Reply:
Ok. Deprecation was was mentioned in one-pager.  will remove.

Reply:
@ijuma and I chatted and we agree that the warning is a bit aggressive in this first release.  We will add a deprecation warning in the README for this first release, and add the ONBUILD warning later.